ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -10,7 +10,6 @@ import BpayLogo from "../../../img/wallet/payment-methods/bancomat_pay.svg";
 import CreditCard from "../../../img/wallet/payment-methods/creditcard.svg";
 import PaypalLogo from "../../../img/wallet/payment-methods/paypal/paypal_logo.svg";
 import SatispayLogo from "../../../img/wallet/payment-methods/satispay-logo.svg";
-import GDOLogo from "../../../img/wallet/unknown-gdo-primary.svg";
```

Comment:
We could remove also the resource from the app, since it's used only here. What do you think?